concent_api/core/tests/utils.py

```diff
@@ -22,6 +22,21 @@
 from golem_messages.factories.tasks import ReportComputedTaskFactory
 from golem_messages.factories.tasks import TaskToComputeFactory
 from golem_messages.message.base import Message
+from golem_messages.message.concents import ForceGetTaskResult
+from golem_messages.message.concents import ForcePayment
+from golem_messages.message.concents import ForceReportComputedTask
+from golem_messages.message.concents import ForceSubtaskResults
+from golem_messages.message.concents import ForceSubtaskResultsResponse
+from golem_messages.message.concents import SubtaskResultsVerify
+from golem_messages.message.tasks import AckReportComputedTask
+from golem_messages.message.tasks import CannotComputeTask
+from golem_messages.message.tasks import ComputeTaskDef
+from golem_messages.message.tasks import RejectReportComputedTask
+from golem_messages.message.tasks import ReportComputedTask
+from golem_messages.message.tasks import SubtaskResultsAccepted
+from golem_messages.message.tasks import SubtaskResultsRejected
+from golem_messages.message.tasks import TaskFailure
+from golem_messages.message.tasks import TaskToCompute
```

Comment:
To organize imports I think better way is just import
```from golem_messages.message import tasks
from golem_messages.message import concents
```
We imports messages like this in whole project

Reply:
@kbeker, for me it doesn't really matter - I would not consider it as a bad practice;)